docs/work-flow.md

```diff
@@ -0,0 +1,176 @@
+
+#### How do I contribute a change to dcos/dcos ?
+
+If a change is only to Enterprise DC/OS, the developer should create the pull request against the Enterprise Repository.
+
+For all contributions, a pull request is against Open Source DC/OS and tested with both Open and
+Enterprise, and we merge the Open Source pull request, and update the reference in Enterprise.
+
+#### What should be my target branch?
+
+All Pull Requests should be against `master` branch, and the bug-fixes should be actively backported to the
+supported versions of DC/OS. The dcos/dcos repo will have the list of actively supported branches, and developers
+should create the back-port pull requests if they have to include the change in the particular version DC/OS.
+
+#### What should I do after I create a pull request against DC/OS?
+
+We use labels, to set the status of a pull-request. All pull requests start at "Work In
+Progress" state. The pull request creator can continue working on this until he/she is ready.
+
+When a pull request is ready for review, the pull request creator is expected to run this command.
+
+```
+@mesosphere-mergebot label ready for review
+```
+
+The above command will set the label of the PR to "Ready For Review" and it is crucial,
+as it will now fall under the radar of Mergebot to monitor for status updates to the PR.
+
+And also, do update to Enterprise reference using the command:
+
+```
+@mesosphere-mergebot bump-ee
+```
+
+The will create a Pull Request against the dcos-enterprise repository and share a link to the pull-request in the github comments.
+
+a. If your changes require Enterprise only addition, you are expected to update the pull request thus created, with your changes.
+If the pull request created has a branch like `mergebot/dcos/1.11/42`
+
+```
+git clone git@github.com:mesosphere/dcos-enterprise.git
+git pull origin
+git checkout -b mergebot/dcos/1.11/2909 origin/mergebot/dcos/1.11/42
+# make your changes to this branch
+git push origin mergebot/dcos/1.11/42
+```
+
+b. If you are an open source contributor, no EE contributions is required; the Bump-EE PR only
+   serves to test and update the reference to the SHA, when your open source contribution is merged.
+
+To re-iterate, with every PR, when it ready for review, developers have to do these two steps
+
+```
+1. @mesosphere-mergebot label ready for review
+```
+
+and
+
+```
+2. @mesosphere-mergebot bump-ee
+```
+
+#### How do I get my reviews for my pull request?
+
+We @mention people the comments on the Pull Request, to get their attention to review a pull request.
+
+Open source contributors can expect that the committers are watching the repo,
+and they will @mention the correct reviewer or will review the PR by themselves.
+
+TODO (skumaran) : DCOS_OSS-1906 - Github CODEOWNERS for dcos/dcos
+
+We expect two approvals at the last commit of your PR (HEAD of your branch), and
```

Comment:
Instead of "we expect", let's be clear and say something like "Mergebot requires".